emulation/src/main.cc

```diff
@@ -7,6 +7,7 @@
 #include "PowerKeys.h"
 #include "Identifiers.h"
 #include <cstdio>
+#include <string>
 using std::cout;
 using std::endl;
```

Comment:
Unrelated to this PR but I feel like this should be driven by a testing framework such as [Google Test](https://github.com/google/googletest)

Reply:
ill look this up